modules/custom-status/custom-status.php

```diff
@@ -669,7 +681,7 @@ public static function delete_custom_status( int $status_id ): bool|WP_Error {
 
 		$old_status_slug = $old_status->slug;
 
-		if ( self::is_restricted_status( $old_status_slug ) ) {
+		if ( self::is_restricted_status( $old_status_slug ) && ! self::is_status_banned_from_slug_changes( $old_status_slug ) ) {
```

Comment:
This code didn't seem to work for me - I can delete Draft and Pending Review statuses right now, in fact all statuses can be deleted until the last throws an error:

![delete-all-statuses](https://github.com/user-attachments/assets/4c308e76-3494-42e3-9026-887f60ebcec8)

Maybe this is supposed to be:

```suggestion
		if ( self::is_restricted_status( $old_status_slug ) || self::is_status_banned_from_slug_changes( $old_status_slug ) ) {
```

Reply:
Yip, accidentally flipped the signs. I tested everything, and then did a bit of cleanup and a bad copy paste went in